llvm/lib/Target/AIE/AIEWawRegRewriter.cpp

```diff
@@ -137,6 +152,27 @@ class AIEWawRegRewriter : public MachineFunctionPass {
   /// removed for future replacement strategies, i.e. block wl4, wh4, y2 if X4
   /// is used.
   void addAliasRegs(BitVector &BlockedPhysRegs, const MCPhysReg PhysReg) const;
+
+  /// \return blocked GPR registers of the \p MF .
+  SmallVector<MCPhysReg, 40> getBlockedGPRs(const MachineFunction &MF) const;
```

Comment:
Perhaps put the type in a 'using' declaration to have a single point of definition